src/coreclr/pal/inc/pal.h

```diff
@@ -1896,6 +1896,9 @@ typedef union IMAGE_ARM64_RUNTIME_FUNCTION_ENTRY_XDATA {
 
 #define CONTEXT_ALL (CONTEXT_CONTROL | CONTEXT_INTEGER | CONTEXT_FLOATING_POINT | CONTEXT_DEBUG_REGISTERS)
 
+#define CONTEXT_HWCAP_LSX 0x10
```

Comment:
This should not have HWCAP in the name (notice that no other CONTEX_ flags do not have HWCAP in the name either).